src/core/imap/MCIMAPSession.cpp

```diff
@@ -996,6 +996,13 @@ static uint64_t get_mod_sequence_value(mailimap * session)
     return mod_sequence_value;
 }
 
+String * IMAPSession::sendCustomCommand(String * command)
```

Comment:
s/sendCustomCommand/customCommand/
Also, you need to add an ErrorCode parameter.